src/mapping.ts

```diff
@@ -1,22 +1,15 @@
-import { NewGravatar, UpdatedGravatar } from '../generated/Gravity/Gravity'
-import { Gravatar } from '../generated/schema'
+import { cosmos } from "@graphprotocol/graph-ts";
+import { Reward } from "../generated/schema";
 
-export function handleNewGravatar(event: NewGravatar): void {
-  let gravatar = new Gravatar(event.params.id.toHex())
-  gravatar.owner = event.params.owner
-  gravatar.displayName = event.params.displayName
-  gravatar.imageUrl = event.params.imageUrl
-  gravatar.save()
-}
+export function handleReward(eventData: cosmos.EventData): void {
+  const height = eventData.block.header.height;
+  const amount = eventData.event.attributes[0].value;
+  const validator = eventData.event.attributes[1].value;
+
+  let reward = new Reward(`${height}-${validator}`);
```

Comment:
What about multiple rewards for the same validator at one height. Is it a possible scenario?

Reply:
Good point. Based on this https://docs.cosmos.network/master/modules/distribution/04_messages.html#msgwithdrawdelegatorreward withdrawing delegation rewards leaves the rewards pool empty, so I guess we can assume you cannot withdraw delegations multiple times within a height.

Reply:
Alright, let's keep it the way it is 👍